bool arrival
---

bool arrival_granted